source/jormungandr/tests/tests_mechanism.py

```diff
@@ -83,19 +84,27 @@ class AbstractTestFixture(unittest.TestCase):
     (the setup() and teardown() methods are called once at the initialization and destruction of the integration tests)
     """
 
+    @classmethod
+    def _get_zmq_socket_name(cls, kraken_name):
+        return "ipc:///tmp/{uid}_{name}".format(uid=cls.uid, name=kraken_name)
```

Comment:
okay, if I understand, this is the new **ipc** that replace `"ipc:///tmp/" + name;` for tests
:+1:

Reply:
it is indeed !